packages/familie-backend/src/auth/tokenUtils.ts

```diff
@@ -87,5 +90,10 @@ export const hasValidAccessToken = (req: Request, key = tokenSetSelfId) => {
     if (!tokenSet) {
         return loggOgReturnerOmTokenErGyldig(req, key, false);
     }
-    return loggOgReturnerOmTokenErGyldig(req, key, new TokenSet(tokenSet).expired() === false);
+    return loggOgReturnerOmTokenErGyldig(req, key, erUtgått(tokenSet) === false);
 };
+
+// kallkjedene kan ta litt tid, og tokenet kan i corner-case gå ut i løpet av kjeden. Så innfører et buffer
+// på 2 minutter.
+const erUtgått = (tokenSet: any): boolean =>
+    new TokenSet(tokenSet).expired() || tokenSet.expires_at - 120 - now() < 0;
```

Comment:
Ser ut som orignalkoden til `expired()` kun forholder seg til `expires_in` - som tar bort behovet for vår egen `now()`
```
expired() {
    return this.expires_in === 0;
  }
```

Kanskje justere litt på denne ->
```
erTokenUtgåttEllerUtgårSnart = (tokenSet: TokenSet) => {
return tokenSet.expired() || tokenSet.expires_in <= 120
}
```

Reply:
Husk att vi nog fortsatt må bruke `new TokenSet(tokenSet)` og sjekke `expired()` og expired_in` mot denne. (Kanskje er tilstrekkelig med kun `new TokenSet(tokenSet).expires_in <= 120 `))